### PR TITLE
Documentation Sync with Discourse

### PR DESCRIPTION
**Pull request by deusebio**

### CODE REVIEW

File: docs/how-to/h-enable-encryption.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+* generate external keys for each unit
+
+```
+openssl genrsa -out external-key-0.pem 3072
+openssl genrsa -out external-key-1.pem 3072
+openssl genrsa -out external-key-2.pem 3072
+```
+
+* apply both private keys on each unit, shared internal key will be allied only on juju leader
```
Comment (paulomach) [inline docs/how-to/h-enable-encryption.md:34]:
```suggestion
* apply both private keys on each unit, shared internal key will be applied only on juju leader
```

File: docs/how-to/h-enable-encryption.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
+# How to enable encryption
+
```
Comment (paulomach) [inline docs/how-to/h-enable-encryption.md:2]: Can we add something like this?

> MySQL will enable encrypted connections by default with self generated certificates. Though also by default, connecting clients can disable encryption by setting the connection ssl-mode as disabled.
> When related with the `tls-certificates-operator` the charmed operator for MySQL will require that every client connection (new and running connections) use encryption, rendering an error when attempting to establish an unencrypted connection.

File: docs/how-to/h-manage-app.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+Also extended permissions can be requested using `mysql-root` edpoint:
+```shell
+juju relate mysql-k8s:mysql-root wordpress-k8s
+```
+
+
+## Rotate applications password
+
+To rotate the passwords of users created for related applications, the relation should be removed and related again. That process will generate a new user and password for the application.
```
Comment (paulomach) [inline docs/how-to/h-manage-app.md:33]:
```suggestion
To rotate the passwords of users created for related applications, the relation should be removed and related again. That process will generate a new user and password for the application, while retaining the requested database and data.
```

File: docs/how-to/h-manage-units.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
+# How to deploy and manage units
+
+## Basic Usage
+
+To deploy a single unit of MySQL using its default configuration
+```shell
+juju deploy mysql-k8s --channel edge
```
Comment (paulomach) [inline docs/how-to/h-manage-units.md:7]:
```suggestion
juju deploy mysql-k8s --channel edge --trust
```

File: docs/how-to/h-manage-units.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+```
+
+It is customary to use MySQL with replication. Hence usually more than one unit (preferably an odd number to prohibit a "split-brain" scenario) is deployed. To deploy MySQL with multiple replicas, specify the number of desired units with the `-n` option.
+```shell
+juju deploy mysql-k8s --channel edge -n <number_of_replicas>
```
Comment (paulomach) [inline docs/how-to/h-manage-units.md:12]:
```suggestion
juju deploy mysql-k8s --channel edge --trust -n <number_of_replicas>
```

**Review by paulomach — CHANGES_REQUESTED** [review]
Left some suggestions/fixes here and there

**Review by taurus-forever — APPROVED** [review]

**Review by paulomach — APPROVED** [review]

**Pull request merged**